<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackControls</class>
 <widget class="QDialog" name="PlaybackControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playback Controls</string>
  </property>
  <widget class="QLabel" name="lblPlaybackPosition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:02</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderPlaybackPosition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextTrack">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-skip-forward.png</normaloff>:/icons/media-skip-forward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreviousTrack">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previous</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-skip-backward.png</normaloff>:/icons/media-skip-backward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblRemainingTime">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>-00:01:01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFastForward">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Fast Forward</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-seek-forward.png</normaloff>:/icons/media-seek-forward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-playback-stop.png</normaloff>:/icons/media-playback-stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-playback-pause.png</normaloff>:/icons/media-playback-pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRewind">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rewind</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-seek-backward.png</normaloff>:/icons/media-seek-backward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTrackList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Current track</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Leave playback controls</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnPlay</tabstop>
  <tabstop>btnPause</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnRewind</tabstop>
  <tabstop>btnFastForward</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>cbTrackList</tabstop>
  <tabstop>btnPreviousTrack</tabstop>
  <tabstop>btnNextTrack</tabstop>
  <tabstop>sliderPlaybackPosition</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPause</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRewind</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onRewind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFastForward</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onFastForward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNextTrack</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onNextTrack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPreviousTrack</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onPreviousTrack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPlaybackPosition</sender>
   <signal>sliderReleased()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onSeekToSliderPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTrackList</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PlaybackControls</receiver>
   <slot>onTrackActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackControls</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPlay()</slot>
  <slot>onPause()</slot>
  <slot>onStop()</slot>
  <slot>onRewind()</slot>
  <slot>onFastForward()</slot>
  <slot>onPreviousTrack()</slot>
  <slot>onNextTrack()</slot>
  <slot>onSeekToSliderPosition()</slot>
  <slot>onTrackActivated(int)</slot>
 </slots>
</ui>
